dm/syncer/ddl.go

```diff
@@ -206,6 +206,7 @@ func adjustCollation(tctx *tcontext.Context, ddlInfo *ddlInfo, statusVars []byte
 		for _, tableOption := range createStmt.Options {
```

Comment:
seems we can `defer adjustColumnsCollation` or first `adjustColumnsCollation` then deal with rest logic related to table

Reply:
ok, i think we can first `adjustColumnsCollation` too, i will fix.